### DISCUSSION TITLE
New look of quoted text blocks setting

### DISCUSSION THREAD

**Discussion by s38b35M5** [spacecowboy/Feeder#303] [Q&A] [answered]
Is there a way to revert to the previous look of quoted text blocks (simple italics) instead of the new lighter background "card" style?

**Comment by spacecowboy** [comment, marked as answer]
Nope.

**Reply by s38b35M5** [reply]
Is there any interest in changing that? It shouldn't represent much work, right? Pretty please?🙏

There's no easy way to export and reimport saved articles, and rolling back to an earlier version means uninstalling and losing that content, not to mention all unread content. The change isn't appealing to my taste compared to the seamless look before.

(Was this change related to the "Added support for iframes inside figures" change? I would've skipped if it said it was a major overhaul to the appearance that can't be altered)

**Reply by spacecowboy** [reply]
I rewrote the rendering engine. It was related to that. I always wanted blocks like this but it wasn't technically possible before.

> Is there any interest in changing that? It shouldn't represent much work, right?

Each individual option is of course not much work but combined together, each option massively increases the complexity of the app. And so I try my hardest to avoid additional options.
